doc/man/zen-tx.1

```diff
@@ -1,5 +1,5 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZEN-TX "1" "July 2020" "zen-tx v2.1.0-beta4" "User Commands"
+.TH ZEN-TX "1" "July 2021" "zen-tx v2.1.0-beta4" "User Commands"
```

Comment:
Should be March 2021, I missed to modify the month by mistake. I won't bother changing it anyhow since this should be all overwritten upon release